packer/tools-setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+sudo debuginfo-install -q -y kernel
```

Comment:
Yeah, this should probably go back to `create-image.sh`. It's here now because there is some conflict that occurs when I install it later. I need to get to the bottom of that.

I'm not sure why we need this in the first place. (I just moved it from `create-image.sh`.) Do you know who might know?